docs/conf.py

```diff
@@ -108,7 +108,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'default'
+html_theme = 'classic'
```

Comment:
I've just noticed that this change has forced the template on readthedocs to use the classic template instead of the readthedocs template. How do you want to handle this?
If this is switched back to default it creates a warning when the docs are built. If we switch it to the readthedocs template then it requires a dependency on the `sphinx_rtd_theme` module.